app_dart/lib/src/service/build_status_provider.dart

```diff
@@ -139,31 +137,6 @@ class BuildStatusService {
     }
   }
 
-  /// This function gets called with the last 40 builds fo a given builder ordered
```

Comment:
In the medium term we would like to get the luci-status as an independent microservice relying only in buildbucket, this functionality will be useful there although most likely in a different class.